examples/jaeger-hotrod/README.md

```diff
@@ -0,0 +1,22 @@
+## Jaeger Hot Rod
+
+This demo will use the Jaeger Hotrod app. 
+
+#### Required:
+* Docker - If you run Docker locally, we recommend allowing Docker to use at least 4 GB of RAM in Preferences > Resources.
+
+#### Demo
+```
+docker-compose up -d --build
+``` 
+
+The above command will start the Jaeger Hotrod sample, Jaeger Agent, OpenTelemetry Collector, SITUP, ODFE and Kibana.
+
+After successful start, use the hot rod app which runs at localhost:8080. Now check the traces in kibana which runs at localhost:5601.
+
+You will see two indicies,
+
+* otel-v1-apm-span  - This index will store the raw traces.
```

Comment:
This should be index alias, the set of indices are in the pattern of otel-v1-apm-span-*.